actionpack/lib/action_controller/metal/flash.rb

```diff
@@ -11,8 +11,8 @@ module Flash
     end
 
     module ClassMethods
-      # Creates new flash types. You can pass as many types as you want to create
-      # flash types other than the default <tt>alert</tt> and <tt>notice</tt> in
+      # Creates new flash types. You can pass as many types as you want,
+      # other than the default <tt>alert</tt> and <tt>notice</tt> in
```

Comment:
The "to create flash types" is a lead of for the "other than default", so lets keep this.